api/src/services/agencies_v1/get_agencies.py

```diff
@@ -53,16 +55,39 @@ def get_agencies(
         .where(Agency.is_test_agency.isnot(True))
     )
 
-    if list_params.filters and list_params.filters.active:
-        active_agency_subquery = (
-            _construct_active_inner_query(Agency.agency_id)
-            .union(_construct_active_inner_query(Agency.top_level_agency_id))
-            .subquery()
-        )
+    if list_params.filters:
+        agency_subquery = None
+        if list_params.filters.active:
+            agency_subquery = (
+                _construct_active_inner_query(
+                    Agency.agency_id, [OpportunityStatus.POSTED, OpportunityStatus.FORECASTED]
+                )
+                .union(
+                    _construct_active_inner_query(
+                        Agency.top_level_agency_id,
+                        [OpportunityStatus.POSTED, OpportunityStatus.FORECASTED],
+                    )
+                )
+                .subquery()
+            )
 
-        agency_id_stmt = select(active_agency_subquery).distinct()
+        elif list_params.filters.active is False:
+            agency_subquery = (
+                _construct_active_inner_query(
+                    Agency.agency_id, [OpportunityStatus.CLOSED, OpportunityStatus.ARCHIVED]
+                )
+                .union(
+                    _construct_active_inner_query(
+                        Agency.top_level_agency_id,
+                        [OpportunityStatus.CLOSED, OpportunityStatus.ARCHIVED],
+                    )
+                )
+                .subquery()
```

Comment:
Same as elsewhere, we might just want to treat False as "no filter" to keep the behavior roughly the same. Won't matter much as it's just while the frontend swaps its usage.